### PR TITLE
41.0 release

### PR DESCRIPTION
**Pull request by hellsworth**
## Description

There's a new 41.0 upstream tag, so let's release it!

## Type of change

Please check only the options that are relevant.

- [x] General Maintenance
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

This PR will build the snap to be tested on Impish and Focal vms.

**Test Configuration**:

* OS (please include version):
* Any other relevant environment information:

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by kenvandine — APPROVED** [review]
Looks good

File: snapcraft.yaml [hunk 1]
```diff
@@ -35,9 +35,57 @@ apps:
...
+      - CFLAGS: -Wno-nonnull
+    override-build: |
+      set -eux
+      snapcraftctl build
+      mkdir -p $SNAPCRAFT_PART_INSTALL/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/glib-2.0/
+      cp $SNAPCRAFT_PART_INSTALL/usr/bin/{gio-querymodules,glib-compile-schemas} $SNAPCRAFT_PART_INSTALL/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/glib-2.0/
+    prime:
+      - -usr/bin
+      - -usr/lib/*/glib-2.0
+      - -usr/share
```
Comment (kenvandine) [inline snapcraft.yaml:68]: Drop usr/include from glib as well

**Review by kenvandine — CHANGES_REQUESTED** [review]
Just one comment inline, don't prime usr/include in the glib part either.

File: snapcraft.yaml (hunk 1 above)
Comment (hellsworth) [reply snapcraft.yaml:68]: Good Point. I've unstaged usr/share now from the glib part.

**Comment by hellsworth** [comment]
It's strange to me that if you run this snap (tested in an updated impish vm) with `snap run gnome-clocks`, it runs, you can do things and it's great, but then when you click the red X button to close it, the cli command doesn't return.

@kenvandine this is the case too for the current gnome-clocks in the stable channel in the snap store. Is this a bug?

**Review by kenvandine — APPROVED** [review]
LGTM

**Pull request merged**